unreleased/elasticsearchreceiver-collect-state_update-metrics.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: elasticsearchreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add "discovery" path in client request to collect elasticsearch.cluster_state.* metrics
```

Orient the change entry toward the user

```suggestion
note: "Fix issue where `elasticsearch.cluster_state.*` metrics were not being collected
```

---

Will keep this in mind for future change entries